> Initialized logger
> Exception in thread Malaria Detection:
> Traceback (most recent call last):
>   File "C:\Users\gherz-PC\Anaconda3\envs\py27\lib\threading.py", line 801, in __bootstrap_inner
>     self.run()
>   File "C:\Users\gherz-PC\Desktop\remidiv2\serveus\detector.py", line 57, in run
>     self.system()
> TypeError: system() takes no arguments (1 given)
>  * Debugger is active!
>  * Debugger PIN: 456-115-780
>  * Detected change in 'C:\\Users\\gherz-PC\\Desktop\\remidiv2\\serveus\\detector.py', reloading
